### ISSUE TITLE
Mouse Keys does not work in resonite.

### ISSUE THREAD

**Issue by Talakai** [Yellow-Dog-Man/Resonite-Issues#2976]
### Is your feature request related to a problem? Please describe.

I often use mouse keys often times when doing click intensive activities. I have difficulty holding my hands steady and find it much easier to spam numberpad 5 than hold a mouse steady and click.

However resonite does not recognize mousekeys at all, which is weird windows itself does. Blender does. I did not know it was even possible to ignore mouse keys considering it has been built into computers for decades. (since 1987)

### Describe the solution you'd like

I'd like to have support added for mousekeys in resonite

### Describe alternatives you've considered

N/A

### Additional Context

![image](https://github.com/user-attachments/assets/287a5e9a-5152-4a81-91c6-1e615ffed9cb)

### Requesters

Talakai
AmasterAmaster

**Comment by Talakai** [comment]
Thank you for adding a tag prime. I am still new to github
